### PR TITLE
Update Monitoring design

### PR DESCRIPTION
**Pull request by ccremer**
Most of the monitoring documentation was already written.

Added a graphic that helps the reader understanding the design

### CODE REVIEW

**Review by tobru — APPROVED** [review]

**Comment by tobru** [comment]
I just saw the word `SYN` in the graphic, I'd prefer to have `Project Syn` used.

**Comment by ccremer** [comment]
Done. Thanks!

File: docs/modules/ROOT/pages/explanations/cluster_monitoring.adoc [hunk 1]
```diff
@@ -31,20 +31,24 @@ The result of doing everything again, would be a huge waste of resources both in
 
 For that reason we'll make use of Cluster Monitoring as much as possible.
 
-To be able to change the existing alerting rules we'll bring our own set of rules based on the https://github.com/openshift/cluster-monitoring-operator[cluster-monitoring-operator] (which in turn is based on https://github.com/prometheus-operator/kube-prometheus[kube-prometheus]).
+To be able to change the existing alerting rules bring our own set of rules based on the https://github.com/openshift/cluster-monitoring-operator[cluster-monitoring-operator] (which in turn is based on https://github.com/prometheus-operator/kube-prometheus[kube-prometheus]).
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/explanations/cluster_monitoring.adoc:34]:
```suggestion
To be able to change the existing alerting rules we bring our own set of rules based on the https://github.com/openshift/cluster-monitoring-operator[cluster-monitoring-operator] (which in turn is based on https://github.com/prometheus-operator/kube-prometheus[kube-prometheus]).
```

File: docs/modules/ROOT/pages/explanations/cluster_monitoring.adoc [hunk 2]
```diff
@@ -31,20 +31,24 @@ The result of doing everything again, would be a huge waste of resources both in
...
 These rules we can change and adapt to our liking with Jsonnet.
 The current rules are ignored by creating https://prometheus.io/docs/alerting/latest/alertmanager/#silences[silences] in AlertManager.
 
-Additionally we'll use the https://docs.openshift.com/container-platform/latest/monitoring/monitoring-your-own-services.html[user workload monitoring] feature to add new scrape targets and Prometheus alert rules.
+Additionally we use the https://docs.openshift.com/container-platform/latest/monitoring/monitoring-your-own-services.html[user workload monitoring] feature to add new scrape targets and Prometheus alert rules.
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/explanations/cluster_monitoring.adoc:38]: Well, that was the idea. If this works or not, I can not tell. At least, we have not testet this yet.

**Review by corvus-ch — CHANGES_REQUESTED** [review]
It was a nice catch to pick up the proposel. Ther is one caveat. The user workload monitoring never got tested. So I can not tell if this works or not.

Options to address:
* Drop that part
* Mark it as planned feature
* Have to pages (one explaining effective available features, one containing the ideas/proposal).

**Comment by ccremer** [comment]
> Options to address:
> 
>     * Drop that part
> 
>     * Mark it as planned feature
> 
>     * Have to pages (one explaining effective available features, one containing the ideas/proposal).

I'd vote for dropping in that case. Having a ticket/issue in the backlog is not exactly "planned".

**Review by corvus-ch — APPROVED** [review]

**Pull request merged**